apps/generator/docs/nunjucka-depreciate.md

```diff
@@ -0,0 +1,120 @@
+--
+title: "Depreciation of nunjucks render engine"
+weight: 170
+---
+
+# Migration Guide from nunjucks render engine to react render engine
+
+## Introduction
+
+AsyncAPI Generator is moving away from Nunjucks templates in favor of React templates. This guide will help you migrate your existing Nunjucks templates to React.
+
+## Step-by-Step Migration Guide
```

Comment:
```suggestion
## Step-by-step migration guide
```
Hey @Gmin2 We use sentence case for all the docs headings. Please re-write the rest of the headings in sentence case